Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -130,5 +131,24 @@ public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_
     return self
   }
 % end
+  
+   public func withDocBlokComment(_ text: String) -> Self {
+    let piece = TriviaPiece.docBlockComment("/** \(text) */")
+    return addCommentPiece(commentPiece: piece)
+  }
+  
+   public func withLineComment(_ text: String) -> Self {
+     let piece = TriviaPiece.lineComment("// \(text)")
```

Comment:
This will behave surprisingly if `text` contains a newline because the second line will not have a `//`. I think we should insert `//` in front of every line. As an added bonus, you could make sure that these lines have the same indentation as the current one, by taking a look at the leading trivia after the last `.newline` in the node’s current `leadingTrivia`.

Reply:
Working on this one 🤝